### DISCUSSION TITLE
Castlevania Advance Collection "no sound"

### DISCUSSION THREAD

**Discussion by mauruco** [farmerbb/RED-Project#28] [Q&A] [answered]
Hey guys. Thank you very much for this project.
I have a question, I used to extract the roms from the new "Castlevania Advnace Collection", it worked but the GBA roms are without music or sound.
Could you help me how to make the audio work?

**Comment by MAlexandros** [comment, marked as answer]
Hi there! We're having a discussion about it on https://github.com/farmerbb/RED-Project/issues/26
It seems like nothing can be done for now...
